omero/sysadmins/unix/install-web.txt

```diff
@@ -55,6 +56,9 @@ customization options.
     install-web/install-apache
     install-web/install-nginx
 
+.. note:: Apache is deprecated as of OMERO 5.2.6 and will not be supported for
+    5.3.
+
```

Comment:
I think you mean a reverse proxy, not forward: https://www.quora.com/Whats-the-difference-between-a-reverse-proxy-and-forward-proxy

Other than that I've got nothing to add.